wordpress-mobile/WordPress-iOS
# Aztec: Properly refreshing Format Bar

### To test:
Play around with the Format Bar. Verify that it matches, consistently, the style of the text.

I've also pushed a couple things here and there, to match the Demo Project // remove a TODO. Hope you don't mind!

Needs review: @astralbodies 
Thanks in advance!
